x/qgb/orchestrator/codec.go

```diff
@@ -0,0 +1,28 @@
+package orchestrator
+
+import (
+	types "github.com/celestiaorg/celestia-app/x/qgb/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/std"
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+	"github.com/tendermint/spm/cosmoscmd"
+)
+
+// MakeEncodingConfig is copied here so that we don't have to have an
+//  import cycle. if possible, use cosmoscmd.MakeEncodingConfig
+func MakeEncodingConfig() cosmoscmd.EncodingConfig {
```

Comment:
Aaah this is new, didn't know it existed.
Will add a TODO  for this and open an issue so that we handle it when we want to merge the QGB integration branch in master.